src/main/resources/reference.conf

```diff
@@ -1312,6 +1337,31 @@ resourceTypes = {
     reuseIds = false
     authDomainConstrainable = true
   }
+  snapshot-builder-request = {
+    actionPatterns = {
+      get = {
+        description = "Get or list the snapshot builder request
```

Comment:
This description is missing its closing double quote (ditto throughout `snapshot-builder-request.actionPatterns`) and I think that's causing the checks to fail.